app/scripts/services/tag-appender.js

```diff
@@ -14,6 +14,16 @@ angular.module('angularWidgetInternal')
 
     return function (url, filetype) {
       var deferred = $q.defer();
+      if (filetype === 'js' && window.requirejs && headElement === document.getElementsByTagName('head')[0]) {
```

Comment:
That sounds good to me. The bad thing is that it's going to affect all the other tests, but I guess it's o.k. Let me know if you want me to make it instead.

Thanks to you!